api/controllers/HealthController.js

```diff
@@ -15,4 +15,16 @@ module.exports = {
       version: packageVersion,
     });
   },
+  getDevtoolsHealth: async function (req, res) {
```

Comment:
@itsalaidbacklife new devtools health endpoint, this gives the frontend awareness and lets us properly log the status. 😎